kuka_kr270_support/urdf/kr270r2700_macro.xacro

```diff
@@ -0,0 +1,216 @@
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_2.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_3">
+      <inertial>
+        <origin xyz="0.18842 0.18344 -0.042799" rpy="0 0 0"/>
+        <mass value="710.03"/>
+        <inertia ixx="11.887" ixy="-0.12154" ixz="-1.3604" iyy="98.805" iyz="-0.056505" izz="96.251"/>
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_3.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_3.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_4">
+      <inertial>
+        <origin xyz="0.27146 -0.007326 5.2775E-05" rpy="0 0 0"/>
+        <mass value="173.73"/>
+        <inertia ixx="1.8001" ixy="-0.18515" ixz="0.00051232" iyy="5.514" iyz="0.00070469" izz="6.3498"/>
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_4.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_4.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_5">
+      <inertial>
+        <origin xyz="0.04379 0.025984 3.5491E-07" rpy="0 0 0"/>
+        <mass value="72.17"/>
+        <inertia ixx="0.3938" ixy="-0.085332" ixz="1.7223E-06" iyy="0.68945" iyz="-7.0292E-06" izz="0.67292"/>
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_5.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_5.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_6">
+      <inertial>
+        <origin xyz="-0.017956 -1.5237E-05 0.00015484" rpy="0 0 0"/>
+        <mass value="6.3154"/>
+        <inertia ixx="0.031746" ixy="1.7673E-07" ixz="-6.6558E-06" iyy="0.016686" iyz="1.4304E-07" izz="0.016723"/>
+      </inertial>
+      <visual>
+        <origin xyz="0.240 0 0 " rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_6.dae"/>
+        </geometry>
+        <xacro:material_kuka_pedestal />
```

Comment:
```suggestion
        <xacro:material_kuka_black />
```

Reply:
T.b.h., I don't see any difference when I change this colors. Are you sure that they are used at all? It seems like the `.dae` files already contain the color information.

Reply:
They are only used for the collision model it seems. So some robots here actually omit them.

Reply:
Done in 492fd2c72c8306d27eb2032a7526703bf972684b